src/Components/FormRadioFieldSet.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { ValidationState } from '../Models/PatientFieldData';
+import ValidationMessage from './ValidationMessage';
+
+import './FormRadioFieldSet.scss';
+
+function FormRadioFieldSet({ children, labelText, property, isRequired, validationState }) {
+  return (
+    <fieldset
+      className={classNames('usa-fieldset form-radio-field-set', {
+        'form-radio-field-set--error': validationState === ValidationState.ERROR,
+        'form-radio-field-set--success': validationState === ValidationState.FIXED,
+      })}
+    >
+      <div className="form-radio-field-set__background">
+        <label className={classNames('usa-label', { 'usa-label--required': isRequired })} htmlFor={property}>
+          {labelText}
+        </label>
+        {children}
```

Comment:
I changed the `formRadios` prop to `children` - this allows you to pass in those elements as nested elements of the FormRadioFieldSet tag, which I think is cleaner than having to put those elements into an array to pass in as an attribute.